server/data/statementsClientTypes.ts

```diff
@@ -8,3 +8,18 @@ type StatementSummary = {
   isOverdue: boolean
   status: string
 }
+
+type Statement = {
+  id?: number
+  bookingId?: number
+  incidentDate?: Date
+  lastTrainingMonth?: number
+  lastTrainingYear?: number
+  jobStartYear?: number
+  statement?: string
+  submittedDate?: Date
+  name?: string
+  reporterName?: string
+}
+
+type AdditionalComments = Array<{ additionalComment: string; dateSubmitted: Date }>
```

Comment:
Good use of generics but it might be better to create a type for the a singular AdditionalComment as it might be more reusable. 

rather than `Array<AdditionalComment>` you can also just do `AdditionalComment[]`

Reply:
agreed. done